libs/tdigest/build.gradle

```diff
@@ -23,6 +23,7 @@ apply plugin: 'elasticsearch.publish'
 
 dependencies {
   api project(':libs:elasticsearch-core')
+  api "org.apache.lucene:lucene-core:${versions.lucene}"
```

Comment:
For `Accountable` and `RamUsageEstimator`

Reply:
:+1: